module/API/Measurer.h

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file Measurer.h
+ * @brief 計測に用いる関数をまとめたラッパークラス
+ * @author keiya121
+ */
+
+#ifndef MEASURER_H
+#define MEASURER_H
+
+#include <algorithm>
+#include "ev3api.h"
+#include "ColorSensor.h"
+#include "SonarSensor.h"
+#include "Motor.h"
+
+class Measurer {
+ public:
+  /**
+   * コンストラクタ
+   */
+  Measurer();
+
+  /**
+   * 明るさを取得
```

Comment:
```suggestion
   * @brief 明るさを取得
```
こんな書き方にしてほしい
[参考記事](https://qiita.com/inabe49/items/23e615649e8539d857a8)

Reply:
レビューありがとうございます。記事を参考にコメントの書き方を変更します。